lib/src/components/Surfaces/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { cn } from "@/utils/cn";
+import { AccordionProps } from "./Accordion.types";
+import { accordionVariants } from "./variants";
+import { twMerge } from "tailwind-merge";
+const Accordion = <C extends React.ElementType = "div">({
+  component,
+  children,
+  classes,
+  defaultExpanded = false,
+  expanded,
+  disabled,
+  disableGutters,
+  square,
+  onChange,
+  id = `accordion-${Math.random().toString(36).substr(2, 9)}`, // Generate a unique id if not provided
+  slots,
+  ref,
+  ...rest
+}: AccordionProps<C>) => {
+  const Component = component || "div";
+  //check if state is controlled or uncontrolled
+  const isControlled = expanded !== undefined;
+  //states for expanded or not
+  const [isExpanded, setIsExpanded] = React.useState(defaultExpanded);
+  // Determine initial open state for uncontrolled mode
+  let isOpen: boolean = isExpanded === true || defaultExpanded === true; // Support both boolean and string for expanded
+  if (isControlled) {
+    if (typeof expanded === "boolean") {
+      isOpen = expanded;
+    } else if (typeof expanded === "string") {
+      isOpen = expanded === id;
+    } else {
+      isOpen = false;
+    }
+  }
+  //handel expansion
+  const handleExpansion = (event: React.SyntheticEvent) => {
+    if (disabled) return;
+    if (isControlled) {
+      if (typeof expanded === "string") {
+        // Pass the next expanded value (panel id or false)
+        onChange?.(event, expanded === id ? false : true);
+      } else {
+        onChange?.(event, !expanded);
+      }
+    } else {
+      setIsExpanded((prev) => (onChange?.(event, !prev), !prev));
+    }
+  };
+  const rootClass = twMerge(
+    "w-100",
```

Comment:
Tailwind’s scale is `w-0` to `w-96` to `w-full`, not `w-100`. `w-100` is invalid.
Change:
Use `w-full` for 100% width:
```suggestion
    "w-full",
```

Reply:
![Screenshot 2025-06-15 172734](https://github.com/user-attachments/assets/82b9bdf2-9efd-4461-acd0-c3f32df549a1)

Reply:
![image](https://github.com/user-attachments/assets/43d9b2b3-2830-4ce5-983c-21cbb3478f0f)
when used w-full

Note: to avoid affecting in external projects, I made the change only in the story
![Screenshot 2025-06-15 172734](https://github.com/user-attachments/assets/73b58f2b-f02f-4374-9812-0b15d1d2f01b)